File: apis/scratch/python/explore.py
from twilio.rest import Client

client = Client("sid", "token")


for msg in client.messages.list():
    print(f"Deleting {msg.body}")
    msg.delete()

#msg = client.messages.create(
#    to="+18328684797",
#    from_="+18057930079",
#    body="hello from python",
#)

#print(f"Created a new message: {msg.sid}")

File: apis/scratch/javascript/explore.js
const Twilio = require("twilio");
const client = new Twilio("sid", "token");

client.messages
    .list()
    .then(messages => console.log(`The most recent message is ${messages[0].body}`))
    .catch(err=>console.error(err));

console.log('Gathering your message log');
